packages/@react-spectrum/picker/docs/Picker.mdx

```diff
@@ -0,0 +1,349 @@
+    {id: 6, name: 'Penguin'},
+    {id: 7, name: 'Snake'},
+    {id: 8, name: 'Turtle'},
+    {id: 9, name: 'Wombat'}
+  ];
+
+  return (
+    // Dynamic Items
+    <Picker label="Pick an animal" items={options} onSelectionChange={selected => console.log(selected)}>
+      {item => <Item>{item.name}</Item>}
+    </Picker>
+  );
+}
+```
+
+### Internationalization
+To internationalize a Picker, a localized string should be passed to the `children` of each `Item`.
+For languages that are read right to left (e.g. Hebrew and Arabic), the layout of the Picker is flipped.
+
+## Labeling
+Picker can be labeled using the `label` prop. If no label is provided, an alternative text label must be provided to identify the control for accessibility. This should be added using the `aria-label` prop. If the Picker is a required field, the `isRequired` and `necessityIndicator` props can be used to show a required state.
+
+```tsx example
+<Picker label="Choose frequency" isRequired necessityIndicator="icon" marginEnd="10px">
+  <Item uniqueKey="rarely">Rarely</Item>
+  <Item uniqueKey="sometimes">Sometimes</Item>
+  <Item uniqueKey="always">Always</Item>
+</Picker>
+<Picker label="Choose frequency" isRequired necessityIndicator="label" marginEnd="10px">
+  <Item uniqueKey="rarely">Rarely</Item>
+  <Item uniqueKey="sometimes">Sometimes</Item>
+  <Item uniqueKey="always">Always</Item>
+</Picker>
+<Picker label="Choose frequency" necessityIndicator="label">
+  <Item uniqueKey="rarely">Rarely</Item>
+  <Item uniqueKey="sometimes">Sometimes</Item>
+  <Item uniqueKey="always">Always</Item>
+</Picker>
+```
+
+## Selection
+Setting a selected option can be done by using the `defaultSelectedKey` or `selectedKey` prop. The selected key corresponds to the `uniqueKey` of an item. See <a href="#events" title="Events">Events</a> for more details on selection events.
+
+```tsx example
+function Example() {
+  let options = [
+    {name: 'Koala'},
+    {name: 'Kangaroo'},
+    {name: 'Platypus'},
+    {name: 'Bald Eagle'},
+    {name: 'Bison'},
+    {name: 'Skunk'}
+  ];
+  let [animal, setAnimal] = React.useState("Bison");
+
+
+  return (
+    <>
+      <Picker
+        label="Pick an animal (uncontrolled)"
+        items={options}
+        defaultSelectedKey="Bison"
+        marginEnd="20px">
+        {item => <Item uniqueKey={item.name}>{item.name}</Item>}
+      </Picker>
+
+
+      <Picker
+        label="Pick an animal (controlled)"
+        items={options}
+        selectedKey={animal}
+        onSelectionChange={selected => setAnimal(selected)}>
+        {item => <Item uniqueKey={item.name}>{item.name}</Item>}
+      </Picker>
+    </>
+  );
+}
+```
+## Sections
+Picker supports sections in order to group options. Sections can be used by wrapping groups of Items in a `Section` component. Each `Section` takes a `title` and `uniqueKey` prop.
+
+### Static Items
+```tsx example
+<Picker label="Pick an animal">
+  <Section uniqueKey="Animals" title="Animals">
```

Comment:
You could remove the `uniqueKey` here, since it's static. React will assign a key automatically, and since it's not used for selection a manual key isn't needed.